frontend/src/bundles/common/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Typography } from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    userFullName: string;
+    url?: string;
+    size?: 'small' | 'medium';
+};
+
+const Avatar: React.FC<Properties> = ({
+    url,
+    size = 'small',
+    userFullName,
+}) => {
+    const avatarClasses = getValidClassNames(styles.avatar, styles[size]);
+
+    const [userName, userSurname] = userFullName.split(' ');
+    const firstLetterIndex = 0;
+    const userNameAbbreviation = userSurname
+        ? `${userName[firstLetterIndex].toUpperCase()}${userSurname[
+              firstLetterIndex
+          ].toUpperCase()}`
+        : `${userName[firstLetterIndex].toUpperCase()}`;
```

Comment:
I saw the same code. Please move it to general helpers and use it

Reply:
It's from one common components - Avatar. All guys that create chat parts (header, chats list, messages list, company info on chat page) use  this 1 avatar components. So it's common part for us but it's same component. 
I will remove it to helpers, maybe someone else will be needed abbreviation.

Reply:
Oh, I see. It is already implemented by mobile team and is present in shared helpers.